### DISCUSSION TITLE
寒冬将至

### DISCUSSION THREAD

**Discussion by shenweiyan** [shenweiyan/Digital-Garden#81] [2.1-乱弹]
经济不景气，就像一股寒风，吹得人心颤颤，市场更是进入严冬，就业、消费和投资都变得如履薄冰。

从最近身边的很多人身上，由于大环境不景气带来的裁员比比皆是，IVD 行业、生信行业订单以肉眼可见的速度越来越少，小公司首先扛不住开启裁员，大公司从考核到绩效各种手段轮番上阵。

<img src='https://gi.weiyan.tech/2024/07/you-are-fired.webp' width=600>

好一点的公司 N+1 赔偿，但凤毛麟角；次一点的公司礼仪性的赔偿 1、2 个月的工资；更多的是领完当月工资走人；更有甚者，老板跑路，变相裁人的 ......

裁人失业年年有，今年的暴风雨可能会更加猛烈些。铁打的营盘，流水的兵，除了比比谁更卷，还是要多思考一下自身的选择和规划。

**Comment by obaby** [comment]
更加艰难了

**Reply by shenweiyan** [reply]
是的，大环境越来越不景气
